src/main/java/seedu/exercise/logic/commands/ScheduleRegimeCommand.java

```diff
@@ -50,11 +60,23 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         schedule(model, toSchedule);
-
+        assert(toSchedule != null);
```

Comment:
the assertion here is kinda useless since there is already a check for the null value in line 58. No `null toSchedule` will pass the if statement above

Reply:
resolved in latest